src/ott/solvers/linear/sinkhorn_lr.py

```diff
@@ -344,12 +377,11 @@ def __call__(
     Returns:
       The low-rank Sinkhorn output.
     """
-    assert ot_prob.is_balanced, "Unbalanced case is not implemented."
     initializer = self.create_initializer(ot_prob)
     init = initializer(ot_prob, *init, rng=rng, **kwargs)
     return run(ot_prob, self, init)
 
-  def _lr_costs(
+  def _get_xis(  # TODO(michalk8): rename
```

Comment:
i though `cost_q` was a bit more self-explaining than `xi_1`, not completely convinced by this new notation! I know it maps better to the paper, but i don't think we should necessarily follow all their notations.

Reply:
I agree!

> TBH i am not a fan of the xi notation, nor xi_1 because I think it has little intuitive meaning, we're losing connection to _q,r,c and also the notion that those are cost matrices. I am fine considering cost matrices rescaled by gamma though. Maybe ga_c ? ga_c_q ?

If you're okay with them being scaled by gamma, will rename!